merge_checks/merge_checks_runner.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+from dataclasses import dataclass
+from enum import Enum
+
+from merge_checks.commit_checks import get_commit_checks_result
+from merge_checks.commit_status_setter import set_commit_status
+
+
+@dataclass
+class Commit:
+    repository: str
+    commit_sha: str
+    token: str
+
+
+@dataclass
+class Status:
+    status_name: str
+    details_url: str
+
+
+class State(Enum):
+    pending = "pending"
+    success = "success"
+    failure = "failure"
+    error = "error"
+
+
+def get_commit_and_status() -> tuple[Commit, Status]:
+    commit = Commit(
+        repository=os.environ["GITHUB_REPOSITORY"],
+        commit_sha=os.environ["GITHUB_SHA"],
+        token=os.environ["TOKEN"],
```

Comment:
```suggestion
        token=os.environ["GITHUB_TOKEN"],
```

Reply:
Changed in 876ca38f96b7a26774737036eda6c989091915bd.